examples/gno.land/p/demo/mux/query.gno

```diff
@@ -0,0 +1,51 @@
+package mux
+
+import "strings"
+
+type UrlQuery map[string][]string
+
+/*
+Query() returns a full map of key-listValue from request
+to get a list of params by key, use: req.Query()["key"]
+to get single param by key, use: req.Query().Get("key")
+*/
+func (r *Request) Query() UrlQuery {
+	urlQueries := UrlQuery{}
+	reqParts := strings.Split(r.Path, "/")
```

Comment:
i think that you're not supporting having a slash in one of the query parameters:

`gno.land/p/demo/foo?a=b&c=d/e`